### DISCUSSION TITLE
About closed discussion

### DISCUSSION THREAD

**Discussion by BECATRUE** [snu-quiqcl/qiwis#102] [Ideas]
We have a lot of discussions, so it makes us confused.

How about closing resolved discussions, except for announcements?

**Comment by BECATRUE** [comment]
Please your opinion freely.

@kangz12345 @Ahrrri

**Comment by kangz12345** [comment]
Oh, I didn't know we can close discussions haha
I want to close them too.

**Comment by BECATRUE** [comment]
I closed some discussions. Please check it.
